distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc

```diff
@@ -1,8 +1,6 @@
-
 ==== Catalog REST Endpoint
```

Comment:
Can you add a blank line on line one as a buffer? It's in case whatever file is placed before this has no blank line at the end.